detox/local-cli/test.js

```diff
@@ -320,9 +328,13 @@ module.exports.handler = async function test(program) {
   }
 
   function clearDeviceRegistryLockFile() {
-    const lockFilePath = environment.getDeviceLockFilePath();
-    fs.ensureFileSync(lockFilePath);
-    fs.writeFileSync(lockFilePath, '[]');
+    const lockFilePathIOS = environment.getDeviceLockFilePathIOS();
+    fs.ensureFileSync(lockFilePathIOS);
+    fs.writeFileSync(lockFilePathIOS, '[]');
+
+    const lockFilePathAndroid = environment.getDeviceLockFilePathAndroid();
```

Comment:
What was the motivation to create two separate lock files for each platform? Theoretically, we could use only one, so why did you discard that option — what are advantages and disadvantages?

Reply:
I'm picturing an exotic use case where on a CI two detox instances run multiple testing for both iOS and Android separately... a single file would have them mutually exclude access to that file, though really they are not fighting over the same resources.

The overhead of having another file is scarce... why not, then? Only caveat is that if we ever allow for multi-platform runs using a single detox instance, it wouldn't work. But... yeah.

Reply:
1. Mmmmm, okay, but it would exclude access for less than a second, I believe — just at the moment, when some of them stringifies JSON array and writes it back to the file.
2. Okay, if you want to keep them separate, why do you clean both lock files regardless of what platform you do run on?? That confuses me, to be honest.